react/src/functions.tsx

```diff
@@ -228,7 +228,7 @@ export function getDatasetFields(dataset: Dataset): Field[] {
         },
         {
             title: "Permission Groups",
-            value: dataset.group_code.join(", "),
+            value: dataset?.group_code?.join(", "),
```

Comment:
Please make sure there are no changes to the `react/` subdirectory (unless needed for features of this PR). You can revert them with `git checkout origin/master react`